### DISCUSSION TITLE
My First GitHub account has been suddenly "flagged" and hidden from public view

### DISCUSSION THREAD

**Discussion by apipoledeng** [community/community#114493] [New to GitHub]
Hi everyone,
Three weeks ago, my GitHub account https://github.com/grithaq was unexpectedly suspended for reasons unknown to me. Despite repeatedly reaching out to GitHub's support team with reinstatement requests, I've yet to receive a response. As a result, my account remains inaccessible to the public.

Any help or advice would be greatly appreciated as I navigate through this challenging situation.

**Comment by nunenuh** [comment]
Dear GitHub Support,

I am following up on our previous support request concerning the accessibility issue with the account of one of our crucial team members, [grithaq](https://github.com/grithaq). It has been over three weeks since the initial ticket was submitted by the account holder, and we have yet to receive a response or update on the situation. This delay is significantly hindering our project progress and team collaboration.

To provide additional context, the account [apipoledeng](https://github.com/apipoledeng) is another account associated with @grithaq. Understanding the connection between these accounts may be relevant to resolving the current access issues.

We are very concerned about the lack of communication and are eager to resolve whatever issues might be causing the restriction or suspension of @grithaq's account. The ability to collaborate and access each other's work is crucial for our ongoing projects. We kindly ask for an expedited review of this situation.

We are fully prepared to comply with any required steps or provide further information necessary to expedite the resolution of this issue. Our team greatly appreciates your assistance and looks forward to a swift response so we can continue our work without further disruptions.

Thank you for your understanding and prompt attention to this matter.

Best regards,
Lalu Erfandi Maula Yusnu

**Comment by ebysofyan** [comment]
Dear Github Support,

I am one of those directly involved in collaborating with @grithaq. I believe your workflow to suspend the user has some hidden bug, since no early warning received by the user. If it related to term of use, please give us response with clear reasons and further actions to take. We will preparing the best effort to make the account comply with term and conditions.

To be honest, this suspension will going to slowing down our progress since we cannot mention @grithaq directly. It make him unaware about the changes. Hopefully, this issue resolved soon as I believe this is only miss information.

Thanks for your attention

Best regards,
Eby Sofyan

**Comment by ghostinhershell** [comment]
Hi @apipoledeng and all, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

For those in this thread who may have already submitted a ticket, the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by mcmerritt13** [comment]
> Hi everyone, Three weeks ago, my GitHub account https://github.com/grithaq was unexpectedly suspended for reasons unknown to me. Despite repeatedly reaching out to GitHub's support team with reinstatement requests, I've yet to receive a response. As a result, my account remains inaccessible to the public.
> 
> Any help or advice would be greatly appreciated as I navigate through this challenging situation.

Were you able to resolve this situation? I'm in the same boat. Made an account and found my account was flagged three days later...
